server/routes/user.js

```diff
@@ -10,19 +11,24 @@ function create(req, res, next) {
 }
 
 function get(req, res, next) {
-  const id = req.params.id;
-
-  return User.findById(id)
+  const where = { id: req.params.id };
+  const include = [
+    {
+      model: Office,
+      attributes: { exclude: ['createdAt', 'updatedAt'] }
+    }];
```

Comment:
1. It is weirdly/wrongly formatted :angry: 
2. I'm assuming you want end up with User object that has `office` property containing office name? If that is the case why not create virtual field that would get resolved with related `Office > name` value: http://docs.sequelizejs.com/variable/index.html (scroll to VIRTUAL)